llvm/unittests/Target/AMDGPU/AMDGPUUnitTests.cpp

```diff
@@ -221,11 +226,10 @@ static void testAbsoluteLimits(StringRef CPUName, StringRef FS,
 }
 
 TEST(AMDGPU, TestOccupancyAbsoluteLimits) {
-  testAbsoluteLimits("gfx1200", "+wavefrontsize32", 1, 16, 256);
-  testAbsoluteLimits("gfx1200", "+wavefrontsize32,+dynamic-vgpr", 1, 16, 128);
-  testAbsoluteLimits(
-      "gfx1200", "+wavefrontsize32,+dynamic-vgpr,+dynamic-vgpr-block-size-32",
-      1, 16, 256);
+  testAbsoluteLimits("gfx1200", "+wavefrontsize32", false, 1, 16, 256);
+  testAbsoluteLimits("gfx1200", "+wavefrontsize32", true, 1, 16, 128);
+  testAbsoluteLimits("gfx1200", "+wavefrontsize32,+dynamic-vgpr-block-size-32",
+                     true, 1, 16, 256);
```

Comment:
Is dynamic-vgpr-block-size-32 a fixed hardware property, or just a parameter to this feature? Should this consolidate into making this an attribute with a value, with 0 meaning disable?

Reply:
The Shader Programming Guide says it's a "chip-wide config", which I'm interpreting as fixed.

Reply:
I'm reading this differently. "The VGPR block size is configurable to be 16-VGPRs with a maximum allocation of 128 VGPRs per wave" - this is a software configurable parameter, it's not a property of the target. It just has poor configurability unlike other kernel settings